Tests/Fixtures/TestProject/project.yml

```diff
@@ -61,6 +61,7 @@ targets:
       - carthage: Alamofire
       - target: App_watchOS
       - target: iMessageApp
+      - framework: Vendor/SomeFramework.framework
```

Comment:
This doesn't exist. It was just to make sure the framework path worked. I could pass `link: false` and `embed: false` to make sure it doesn't affect the build. Not sure if the project has to compile.

Reply:
Yeah, I've been trying to keep the project buildable. I've actually been thinking of adding some `xcodebuild` commands to the tests to validate that everything still builds